README.md

```diff
@@ -127,6 +127,7 @@ require('lualine').setup {
     },
     ignore_focus = {},
     always_divide_middle = true,
+    always_show_tabs = true,
```

Comment:
I think always_show_tabline will be a better name. because tabline not necessarily shows just the tabs.